src/Interactive/HostProcess/InteractiveHostEntryPoint.cs

```diff
@@ -33,7 +33,9 @@ private static async Task<int> Main(string[] args)
                     Application.Run();
                 });
 
+#pragma warning disable CA1416 // analyzer is warning this API is only available on Windows, but this project only targets windows.
```

Comment:
Using a Debug.Assert sounds more neat than a suppression.

Reply:
This appears to be an analyzer bug actually...

Reply:
Pinging @buyaa-n on this.

Reply:
if the project targets windows it shouldn't warn for calling windows only API, there were MSBuild bug related to version less target platform https://github.com/dotnet/sdk/issues/13494, might be caused from that

Reply:
@buyaa-n The project target both, net5.0-windows **and** net472:

```xml
<TargetFrameworks>net472;net5.0-windows</TargetFrameworks>
```

Reply:
The analzyer is off by default for net472

Reply:
I'm guessing the net472 isn't being counted as "windows" and I guess maybe there's Mono scenarios that apply here.